frontend/src/App.js

```diff
@@ -1,163 +1,204 @@
 import axios from "axios";
+import React, { useState, useEffect } from "react";
+import { BrowserRouter as Router, Switch, Route, Link, withRouter, Redirect } from "react-router-dom";
 import localStorageService from "./services/LocalStorageService";
-import React from "react";
-import {
-    BrowserRouter as Router,
-    Switch,
-    Route,
-    Link, withRouter, Redirect
-} from "react-router-dom";
 
+import About from "./pages/About";
 import Post from "./components/Post";
+import PostPicture from "./components/PostPicture";
 import Login from "./components/Login";
 import Register from "./components/Register"
+import User from "./components/User";
```

Comment:
Can these imports be in alphabetical order? Small detail.

Reply:
Will do!